catalogue/aarch64/README.md

```diff
@@ -0,0 +1,20 @@
+aarch64: Catalogue with base tests for AArch64
+==============================================
+
+Includes tests that exercise the semantics of coherence (for example,
```

Comment:
the semantics of coherence -> that ask questions about the coherence of a system

and store buffering -> , store buffering